.github/workflows/pr-title-checker.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow/action checks if PR titles conform to the Contribution Guidelines
+# For more information see: https://github.com/webex/pr-title-checker
+
+name: pr-title-checker
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - labeled
+      - unlabeled
+    branches:
+      - master
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR title
+        uses: webex/pr-title-checker@main
```

Comment:
https://github.com/webex/pr-title-checker
This Action is forked in Webex, we checked this workflow is trigerring in Embedded app SDK.

https://github.com/webex/embedded-app-sdk/actions/workflows/pr-title-checker.yml